src/ngx_http_lua_shdict.c

```diff
@@ -347,10 +347,13 @@ ngx_http_lua_inject_shdict_api(ngx_http_lua_main_conf_t *lmcf, lua_State *L)
             lua_pushlstring(L, (char *) ctx->name.data, ctx->name.len);
                 /* shared mt key */
 
+            lua_createtable(L, 1 /* narr */, 0 /* nrec */); /* table of zone[i] */
             lua_pushlightuserdata(L, zone[i]); /* shared mt key ud */
+            lua_rawseti(L, -2, 1); /* {zone[i]} */
```

Comment:
I think it's better to replace the magic number index `1` with a named enum name. For example,

```C
enum {
    SHDICT_USERDATA_INDEX = 1
};
```

near the beginning of this `.c` file and then write

```C
            lua_rawseti(L, -2, SHDICT_USERDATA_INDEX); /* {zone[i]} */
```

What do you think?